mastodon/src/main/java/org/joinmastodon/android/fragments/settings/SettingsNotificationsFragment.java

```diff
@@ -319,7 +319,7 @@ private void updateBanner(){
 			bannerText.setText(R.string.notifications_disabled_in_system);
 			bannerButton.setText(R.string.open_system_notification_settings);
 			bannerButton.setOnClickListener(v->openSystemNotificationSettings());
-		}else if(BuildConfig.BUILD_TYPE.equals("fdroidRelease") && useUnifiedPush){
+		}else if(BuildConfig.BUILD_TYPE.equals("fdroidRelease") && !useUnifiedPush){
```

Comment:
The idea was to only show the banner if the user chooses to enabled UnifiedPush, as otherwise the banner does not concern them. This would also hide the banner if UnifiedPush is enabled which won't help with the troubleshooting them the banner is supposed to provide.